redisinsight/ui/src/pages/vector-search/useRedisearchListData.ts

```diff
@@ -27,5 +31,6 @@ export const useRedisearchListData = () => {
   return {
     loading,
     data,
+    stringData,
```

Comment:
Note: To be honest, I'm not quite sure whether I like the idea to return the same data in two different variants (buffer and a string), so I would propose to return only the string representation if that’s what we need. 

However, I've heard a few times that buffer to string conversion is not always safe (something due to size limitation and so on), so I'm not sure whether it's applicable to this use case.

Consider this a note, in case you're not sure either.